src/components/TrackGameOne.js

```diff
@@ -0,0 +1,38 @@
+import Input from './Input.js';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+TrackGameOne.propTypes = {
+  onHandleChange: PropTypes.func,
+  gameData: PropTypes.string
+};
+
+export default function TrackGameOne({ onHandleChange, gameData }) {
+  return (
+    <>
+      <GameNameContainer>
+        <Input
+          name="nameOfGame"
+          style={{ width: '335px' }}
```

Comment:
I wouldn't use inline style but restyle the `Input` component. 
```js
const InputStyled = styled(Input)`
  width: 335px;
`
```
But I'd also be careful about setting fixed widths in general :)